substratevm/src/com.oracle.svm.jfr/src/com/oracle/svm/jfr/JfrRecorderThread.java

```diff
@@ -102,7 +103,9 @@ private void persistBuffers(JfrChunkWriter chunkWriter) {
                 JfrBufferAccess.release(buffer);
 
                 if (shouldNotify) {
-                    Target_jdk_jfr_internal_JVM.FILE_DELTA_CHANGE.notify();
+                    synchronized (Target_jdk_jfr_internal_JVM.FILE_DELTA_CHANGE) {
+                        Target_jdk_jfr_internal_JVM.FILE_DELTA_CHANGE.notifyAll();
```

Comment:
Thanks for fixing that. As we don't have any other choice than to use `synchronized` and `notifyAll` (i.e., the JDK code expects that behavior), please explicitly place `//Checkstyle: stop` before the synchronized and `//Checkstyle: resume` after the `synchronized` block. Then, it should be possible to remove `@SuppressWarnings`.

Reply:
Okay, added the stop/resume comments; thank you!